### ISSUE TITLE
Cannot scan large multisig transaction QR code with Electrum on android

### ISSUE THREAD

**Issue by viperperidot** [spesmilo/electrum#6642]
When trying to scan a QR code for a multisig transaction on Electrum android it only works if it is a small QR code. If you are trying to scan a QR code that is large (which gets produced when you have multiple utxos or addresses in one transaction) it does not work. You hold the phone up to the qr code and it does not scan due to being too large to read I suppose.

I am not sure what the threshold is for the size limit but I have been able to scan smaller multisig transaction qr codes on Electrum android before, once the qr code get's too big the scanner does not read it.

This is an issue for people who use multisig on their mobile device and need to scan transactions in order to sign and broadcast partially signed transactions.

**Comment by SomberNight** [comment]
Are you using Electrum on desktop (Qt GUI) to display the QR code that you scan with Electrum on Android?
What phone do you have? (just interested if it's cheap / camera specs)

We could try increasing the error correction rate in the QR code (but then that decreases the max payload size, so the max tx size that fits).

**Comment by viperperidot** [comment]
Yes, I am using Electrum on desktop with the current version to display the QR code of the partially signed transaction to scan with Electrum on Android.

I am using a cheap phone right now (Samsung Galaxy A11) but I believe I had the same issue when using my Samsung Galaxy S10, if I remember correctly. 

The transaction I was trying to scan was not overly large, it only had 4 utxos.

**Comment by Armantheparman** [comment]
Actually, it might be the case for any device. Large QR codes with electrum (2 UTXO inputs, 1 output, 3 of 5 multisig transaction) don't work on Android phone, and Mac laptop front camera and Mac desktop front camera.

**Comment by viperperidot** [comment]
Hmmm ok, is there a way to fix this? I see it as an issue if qr codes can be generated but not read. Or is this a case where third parties (android camera hardware/firmware?) need to catch up to qr code technology and is out of our hands?

**Comment by Transisto** [comment]
I had it to work today but it was quite a struggle orient it so it could scan.
